tutorials/ph-ug-temp

```diff
@@ -0,0 +1,102 @@
+---
```

Comment:
File must be called `index.md` in a subfolder under `tutorials/`, for example:
`tutorials/studying-land-surface-temp-ug/index.md`